CS342MemberViews/Sources/EmmyThamakaison/LinkedIn.swift

```diff
@@ -0,0 +1,40 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+struct LinkedIn: View {
+    @Environment(\.openURL) var openURL
+    
+    
+    var body: some View {
+        if let linkedInURL {
+            Button {
+                openURL(linkedInURL)
+            } label: {
+                Text("LinkedIn")
```

Comment:
You have a LinkedIn icon in your Resources folder that you might want to remove as it is not used.